import "ydb/library/actors/protos/actors.proto";

package NKikimrGRpcService;

message TEvSubscribeGrpcCancel {
    optional NActorsProto.TActorId Subscriber = 1;
};

message TEvClientLost {
};
